server/scripts/check_sdb_migrations.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -eu
+
+echo $PGUSER > /dev/null
+echo $PGPASSWORD > /dev/null
```

Comment:
This doesn't make sense :smile:

Reply:
This tests that the variables are set, with set -u